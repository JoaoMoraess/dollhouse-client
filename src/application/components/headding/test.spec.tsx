import { render, screen } from '@testing-library/react'

import Headding from '.'

describe('<Headding />', () => {
  it('should render the heading', () => {
    const { container } = render(<Headding />)

    expect(
      screen.getByRole('heading', { name: /Headding/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
