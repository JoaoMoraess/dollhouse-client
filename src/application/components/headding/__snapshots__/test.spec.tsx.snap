// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Headding /> should render the heading 1`] = `
<div>
  <h1
    class="bg-pink-300 inline-block p-2 rounded-lg text-gray-800"
  >
    Headding
  </h1>
</div>
`;
